WeakAuras/WeakAuras.lua

```diff
@@ -1382,7 +1371,7 @@ loadedFrame:SetScript("OnEvent", function(self, event, addon)
       dbIsValid = true
     end
     if dbIsValid then
-      -- run login thread for up to 15 seconds, then defer to dynFrame
+      -- run login thread for up to 15 seconds, then defer to Threads
```

Comment:
the comment here is now wrong

Reply:
I think it's been wrong for a long time i.e. the 15000 we pass into Login is used as a boolean value.